content/webapp/components/Table/Table.tsx

```diff
@@ -1,7 +1,8 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-nocheck
-// Sorry to have done this, but this file is riddled with errors.
-// I'll sort it in another PR.
+// TODO: To fix when we create a new Storybook - this component could actually be deleted
```

Comment:
Because we're strongly thinking of removing it, this component probably throws some warnings about props rendering in the DOM, so if we turn around and keep it, the comments above will need to be removed and the component sorted. The previous comment saying it would be was 3 years old.